jsHelper/spicetifyWrapper.js

```diff
@@ -402,6 +405,11 @@ window.Spicetify = {
 			ButtonPrimary: modules.find(m => m?.render && m?.displayName === "ButtonPrimary"),
 			ButtonSecondary: modules.find(m => m?.render && m?.displayName === "ButtonSecondary"),
 			ButtonTertiary: modules.find(m => m?.render && m?.displayName === "ButtonTertiary"),
+			Snackbar: {
```

Comment:
just to be safe
```suggestion
			Snackbar: {
				...Spicetify.Snackbar,
```

Reply:
why would we want the normal snackbar functions in the react components?

Reply:
https://github.com/spicetify/spicetify-cli/pull/2601#discussion_r1362322493